components/AirTableCard.js

```diff
@@ -112,6 +126,18 @@ export default function AirTableCard({ location }) {
 
   const availabilityStatus = location.availabilityStatus;
 
+  const ageRequirement = getOptionalAdditionalInfo(location.fields.age_requirement);
+  const occupationRequirement = getOptionalAdditionalInfo(location.fields.occupation_requirement);
+  const countyRequirement = getOptionalAdditionalInfo(location.fields.eligible_counties);
+
+  // TODO: We could order these so that they are always in the same spot (e.g. age | occupation | location )
```

Comment:
Open q. I think the design is much nicer as is with no empty sections, but can also see the argument for each piece of data having their own location that potentially gets left empty for the sake of scanability.